app/models/subcase.js

```diff
@@ -228,10 +228,11 @@ export default Model.extend({
 
   latestAgendaItem: computed('latestAgenda', async function() {
     const latestAgenda = await this.get('latestAgenda');
+    latestAgenda.hasMany('agendaitems').reload();
     const latestAgendaItems = await latestAgenda.get('agendaitems');
     const agendaitems = await this.agendaitems;
 
-    return latestAgendaItems.find(item => agendaitems.includes(item))
+    return latestAgendaItems.find(item => agendaitems.includes(item));
```

Comment:
Is `latestAgendaItems` een object hier of een string?
Zo ja: negeer
Zo nee: Dan vind ik `includes()` gevaarlijk met een object.